src/main/java/io/vertx/rabbitmq/RabbitMQConsumer.java

```diff
@@ -66,8 +66,14 @@
    * <p>
    * The operation is asynchronous. When consumption will be stopped, you can by notified via {@link RabbitMQConsumer#endHandler(Handler)}
    */
-  void cancel();
+  Future<Void> cancel();
```

Comment:
if we do have this cancel method with a Future return then we need also to have an `cancel(Handler<AsyncResult<Void>>)` method alternative